recipes/itpminer/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "itpminer" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 4b14a791deaefbe8fbabd1f34c72d79741f6cbe685448cce3ab9441f497d33e1
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - matplotlib
```

Unless you need a PyQt-based GUI, it's better to depend on `matplotlib-base`:
```suggestion
    - matplotlib-base
```

---

Thank you for the suggestion!